libs/npm/core/src/components/menu/MenuGroup.tsx

```diff
@@ -2,10 +2,11 @@ import { Menu as HMenu } from '@headlessui/react';
 import React, { HTMLAttributes } from 'react';
 import { IconType } from 'react-icons/lib';
 
-import { Claim, Role, useKeycloakWrapper } from '../../hooks';
 import { MenuStatus } from '.';
 import * as styled from './styled';
-import { isInViewport } from 'utils';
```

Comment:
For some reason this doesn't work in the package.  You have to use relative paths.